podcasts/New Detail/Custom Order/PlaylistDetailCustomOrderViewController.swift

```diff
@@ -0,0 +1,159 @@
+        changeNavTint(titleColor: AppTheme.colorForStyle(.primaryText01), iconsColor: AppTheme.colorForStyle(.primaryIcon03), backgroundColor: backgroundColor)
+
+        title = L10n.playlistManualEpisodesOrderOption
+        navigationController?.navigationBar.prefersLargeTitles = false
+        navigationItem.largeTitleDisplayMode = .never
+
+        let appearance = UINavigationBarAppearance()
+        appearance.backgroundColor = AppTheme.colorForStyle(.primaryUi01)
+        appearance.titleTextAttributes = [
+            NSAttributedString.Key.foregroundColor: AppTheme.colorForStyle(.primaryText01)
+        ]
+        navigationController?.navigationBar.scrollEdgeAppearance = appearance
+        navigationController?.navigationBar.standardAppearance = appearance
+        navigationController?.navigationBar.sizeToFit()
+    }
+
+    private func setupContent() {
+        tableView = ThemeableTable()
+        view.addSubview(tableView)
+
+        NSLayoutConstraint.activate([
+            tableView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 0),
+            tableView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: 0),
+            tableView.topAnchor.constraint(equalTo: view.topAnchor, constant: 0),
+            tableView.bottomAnchor.constraint(equalTo: view.bottomAnchor, constant: 0)
+        ])
+
+        view.layoutSubviews()
+
+        insetAdjuster.setupInsetAdjustmentsForMiniPlayer(scrollView: tableView)
+    }
+
+    private func registerCells() {
+        tableView.register(PlaylistEpisodePreviewCell.self, forCellReuseIdentifier: PlaylistEpisodePreviewCell.reuseIdentifier)
+    }
+}
+
+extension PlaylistDetailCustomOrderViewController: UITableViewDataSource, UITableViewDelegate {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return episodes.count
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: PlaylistEpisodePreviewCell.reuseIdentifier, for: indexPath) as! PlaylistEpisodePreviewCell
+        let listEpisode = episodes[indexPath.row]
+        cell.set(episode: listEpisode.episode)
+        return cell
+    }
+
+    // MARK: - Editing
+
+    func tableView(_ tableView: UITableView, canEditRowAt indexPath: IndexPath) -> Bool {
+        true
+    }
+
+    func tableView(_ tableView: UITableView, editingStyleForRowAt indexPath: IndexPath) -> UITableViewCell.EditingStyle {
+        .delete
+    }
+
+    func tableView(_ tableView: UITableView, shouldIndentWhileEditingRowAt indexPath: IndexPath) -> Bool {
+        false
+    }
+
+    func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCell.EditingStyle, forRowAt indexPath: IndexPath) {
+        if editingStyle == .delete, let episode = episodes[safe: indexPath.row] {
+            // TODO: Delete episode from playlist in DB here?
```

Comment:
@bjtitus we should decide whether or not add the DB operation to delete the episode from here or pass the episode to the parent vc so the view model can do the DB operation

Reply:
Resolved [eaa4ea4](https://github.com/Automattic/pocket-casts-ios/pull/3520/commits/eaa4ea4d60bf44e73829a639f4c10ebabb335b34)